pkg/content_clean.go

```diff
@@ -0,0 +1,390 @@
+package pkg
+
+import (
+	"bufio"
+	log "github.com/sirupsen/logrus"
+	"github.com/wttech/aemc/pkg/cfg"
+	"io"
+	"io/fs"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+)
+
+const (
+	JcrContentFile     = ".content.xml"
+	JcrMixinTypesProp  = "jcr:mixinTypes"
+	JcrRootPrefix      = "<jcr:root"
+	ContentPropPattern = "^\\s*([^ =]+)=\"([^\"]+)\"(.*)$"
+	NamespacePattern   = "^\\w+:(\\w+)=\"[^\"]+\"$"
+	JcrRoot            = "jcr_root"
+)
+
+func prepare(root string, config *cfg.ConfigValues) error {
+	if config.Content.ParentsBackupEnabled {
+		return doParentsBackup(root, config)
+	}
+	return nil
+}
+
+func BeforeClean(root string, config *cfg.ConfigValues) error {
```

Comment:
basically I am trying to organise code into struct/services... you just implemented set of methods. this style is commonly used in Go community as well but I prefer to have style aligned everywhere.

how about 

```go
package content

type Cleaner struct {

}

func (c Cleaner) Clean(root string) error{

}
```